rclcpp/include/rclcpp/timer.hpp

```diff
@@ -58,6 +58,15 @@ class TimerBase
   void
   cancel();
 
+  /// Return the timer cancellation state.
+  /**
+   * @return true if the timer has been cancelled, false otherwise
```

Comment:
nitpick: we use `\` rather than `@` in our doxygen comments

Reply:
Addressed in the latest commit.